City-of-Helsinki/helsinki-paatokset
# Pp 148 calendar for policymakers

Created a calendar for policymaker page.

**How to test**

Checkout branch and run `make drush-cim` and `make drush cr`. Build the styles in the theme folder by running `npm run dev`. (might have to clear caches after this) Make sure you also have the meeting and policymakers migrations run properly.

Go to some policymaker page i.ex. https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginvaltuusto and scroll down. At the moment there shouldn't be any calendar visible since there aren't any upcoming meetings. 

Find some meetings with the same policymaker in the content and edit them to be in the future form today. Go to the policymaker page again and the calendar should now be visible. Test also that the meetings description works by editing the policymaker node and add some text to the meeting description.

Check that the styles match the [design](https://app.abstract.com/projects/9e324ba2-c4b2-45b5-93d9-92f00c1c688e/branches/master/commits/971a170eb56a625120d522b229ae63438e048be2/files/7bad9819-85bc-4fe1-b2bb-e53b8b35d442/layers/536EE235-341B-41B2-BEF9-5E5399D9ED64?collectionId=ca39c173-6fde-43db-bbe6-8e10d0544a47&collectionLayerId=33e303c2-9320-47eb-ab69-6d6eac9201ca&mode=build) in desktop and mobile view.

At the moment the links to motions are just dummy data as well as the "lisätietoja" column.
